### PR TITLE
Add package: nodejs

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

File: var/spack/repos/builtin/packages/nodejs/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Nodejs(Package):
```
Comment (davydden) [inline var/spack/repos/builtin/packages/nodejs/package.py]: please use AutotoolsPackage

File: var/spack/repos/builtin/packages/nodejs/package.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    homepage = "https://nodejs.org/"
+    url      = "https://nodejs.org/dist/v4.5.0/node-v4.5.0.tar.gz"
+
+    version('4.5.0', '1885586b4b8a2263f77dce27855661e9')
+
+    depends_on('python')
+    depends_on('openssl')
+
+    def install(self, spec, prefix):
```
Comment (davydden) [inline var/spack/repos/builtin/packages/nodejs/package.py]: This must go, see documentation/other packages on which functions to override. I am AFK, can't copy-paste

**Comment by adamjstewart** [comment]
There is already a `node-js` package in Spack.

**Comment by davydden** [comment]
@adamjstewart time to have some mechanics to check package duplicates? :smile:

**Comment by adamjstewart** [comment]
Heh, I'll consider it. Shouldn't be too hard.

**Pull request closed without merging**